src/misc.c

```diff
@@ -664,6 +664,19 @@ setup_win_version(void)
             if(fn_IsWow64Process != NULL)
                 fn_IsWow64Process(GetCurrentProcess(), &is_64bit);
         }
+        if (!is_64bit) {
+          BOOL(WINAPI * fn_IsWow64Process2)(HANDLE, USHORT*, USHORT*);
```

Comment:
Could you please use 4 spaces as elsewhere for the block indentation?

Reply:
should be all good now, must've copied it before setting indents to 4